posthog/api/test/__snapshots__/test_decide.ambr

```diff
@@ -171,7 +171,8 @@
          "posthog_featureflag"."active",
          "posthog_featureflag"."rollback_conditions",
          "posthog_featureflag"."performed_rollback",
-         "posthog_featureflag"."ensure_experience_continuity"
+         "posthog_featureflag"."ensure_experience_continuity",
+         "posthog_featureflag"."usage_dashboard_id"
```

Comment:
Got to be careful of this perf wise! I'd prefer if we didn't select it at all, but ok with this as long as no joins come into the picture 👀

Reply:
Shouldn't need joins. The component on the frontend just needs the id